OpenCDMi/FrameworkRPC.cpp

```diff
@@ -1369,7 +1485,19 @@ namespace Plugin {
 
                     //now handle the configuration
                     if (factory != factories.end()) {
-                        const string configuration(index.Current().Configuration.Value());
+                        string configuration(index.Current().Configuration.Value());
+                        if(system == "WideVine") {
+                            WidevineConfig widevineConfig;
```

Comment:
To make our merging life easier in the future. Could that block be rewritten to some method - it is only for simplify future management of that change